pkg/uhttp/wrapper.go

```diff
@@ -215,6 +232,30 @@ func WithAcceptXMLHeader() RequestOption {
 	return WithHeader("Accept", "application/xml")
 }
 
+func WithContentTypeFormHeader() RequestOption {
```

Comment:
It looks like there are already some helper functions so you could do
```go
func WithContentTypeFormHeader() RequestOption {
	return WithHeader("Content-Type", "application/x-www-form-urlencoded")
}
...
func WithAcceptVndJSONHeader() RequestOption {
	return WithHeader("Accept", "application/vnd.api+json")
}
```
or add new helpers and do something like:
```go
func WithContentType(value string) RequestOption {
	return WithHeader("Content-Type", value)
}
func WithContentTypeFormHeader() RequestOption {
	return WithContentType("application/x-www-form-urlencoded")
}
...
```

Reply:
Yes. Refactoring is always valid. Thanks..!!

Reply:
Fixed